ibm/mas_devops/roles/gencfg_jdbc/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
+
+# 1. Check for undefined properties that do not have a default
+# -----------------------------------------------------------------------------
+- name: "Fail if mas_instance_id is not provided"
+  when: mas_instance_id is not defined or mas_instance_id|length == 0
```

Comment:
Can we change the "when..fail" checks to be "assert..that" checks. Like https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/suite_mustgather/tasks/main.yml#L6

Reply:
Changed

Reply:
@shajeena For this I would add a "when" condition to the assert. Like this:
```
assert: 
  that: mas_application_id is defined and mas_application_id|length != 0
  fail_msg: "mas_application_id property is required"
when: mas_config_scope == "app" or mas_config_scope == "wsapp"
```